templates/ssh/authorized_keys

```diff
@@ -97,7 +97,8 @@ ssh-dss AAAAB3NzaC1kc3MAAACBAP7HlchKGwCCNtqAuVZWMfK6unfWj8EVAe9kIbyU6Pk3fM9LvQlW
 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDiyDl0SgvVHWg3/+NtrVbQQEj2EqXw0YBatK+yC5bG9JI91y9deltM29ey+DDKyMBnpRcSqljp7kPV45d26g4Bca8/GoaH2UCpquCcNz1V3gQ6mPTpCGR41H+QHrDbXoQ3wSf6wpZuh8L8R3kgjCzrdhhkRp0jyoq2UsQZwtyjfjV0s5UQNHWbklSOuxLRNSjlXSNxxFitOQbpw7puecn6avrHW3KRg1lE6j/2ZzmrUEzR7Xruubrj5o/tfRzrkcp8q6u1bPs8eVJP9eL8fV4LheWoYB2MwhUPMNPYsDBG2sIUxuf5hCxP7ANopf/c0GPj3sOplqhsF0baJ0hVsu5wzrjeWrhZ+b4lFtkXlBt3EhvYW+lUP4q9c4bupgXylq54S6+8dKLoOfyb1RRc6drJoD8BGsBzgnPnz7CleqrQ+b9zMQn8rvJXgvrpPmu2vH5bfpeZMHIfbA/IPwtUZvEYaF9rn30EHFIji0vhf71X21AtQ8H4iT6I/JVscUmqsgKm95xa+i20SYvZEcd9p23gITxQjveVEq91GFV7zdaJnx9KwXDnyvvRoJyUwI4yCSG29f+oIZbNq9e19NKcGSzUJCySwCX3lWGQOQy4nQXoC/7Fgp+RlX/gaJ3qrM/u9PaTfppPjbbFWH8PdemoBLk7PqKEGMuWnOz+3rqgXpXWFw== shermdog@puppetlabs.com
 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDMlV83Evx6LhGFJ+hWKekION/ANE0E1gbNSWaonT5dqGjzR/AzUraacr5VT7duvQ25fhOTvn7z/xbIsIFRn0pTN45L1v7sRuErcM3jD8KCoUtyon85+gZlqnapKFbqx2ESAIt9cCqGabr3ITUB9Z10mjzfI1r/NX8C/o/8NwXSIkytpA2OVh8jikaDg2sfLmD7Tt0CVUO3+QWVajY7nrIq42umNSEMWcy8wnLyz1l+WyCcTa6s56ouYtF4U5luRHRQuD3TNeBMu+mipocXFZ8ldcpFs5tOYTxTZv/riX81XaODRysW9JTO59pwJ4zImM+RxgWa9zLIzwAqd852j9PmZnh72y5oIXAq4HYzBN7NIjCRdQntlGWATBhN2c2h+UqevKTF8D3CRFxI5nCuoIE5FeHGySnueez07iCK3MiSjGmU9pbj2vu7DoJGnIyVHzY/rGsPVLHKMyHmPwaKyGXlK4kcksval2u5ayTXeLJfD5Z/w2lhz3Y/os+jBzdo3gr7J3OpyjC4f3HKandX4TE/MGsabz6IxL0VVL0yuztAH5MOi044VCOSNmXleAtB0Ls2HwaYSV3WlNyA+Ro9p01dbARTfkC6mUXftpzxfwbcFKwhW8UMhgFM1b0yKeCbMF1s5qaQBscRXNz7zKHLBe/fXBvuFm87D8m1Th+8Jmm29Q== reid-2014-04-17@puppetlabs.com
 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDQrUV5+yB65w++xOQBi8PvsBGraPkG6/l6i5a3iHDL31M5c3sFMDHYx3SWxo0OIYxlBeY6S2VhMTURlwTDLvQO6Q4IRIPDz61LX15d+EfsMQyveYh53SHEyPsKfjzIYL3CfkUUVp1lvSlJ0IdAflm/1e2ajAVaSUNcut6PkBJ/a0JitpmQktrmx9Gz/7ERR1vC3DdYQvYdutAxYHkrOe5+7/BHM72fKbMtd/3iV3DtLYQW/jnWyfCEbTw64GOty+R7J2TLqHhJffu8ad9tX2q12mPiuOp2Em1DDc2qosBGfTUuWVGmBqzepI79ZzKt561Sf7UMQbBLDN1QIClBURRr michal@placka.local
-ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDvh9zcxG7a6qATxCCzhQXWjtCCsjALwAiUmwamvilDIL+rL3sRVJjm/aixv9Wi3QTaU/529XkAxkxNVMKo93q6j7wk/Lm2sJR/gdbvxY1IYdz0MdPR1iuM2gSXHaSPvCa+sZ/D4S4JZlrpGab8BM2XJNoTlRH3lNbDu65NRMTk/Ja6z5wZMK8w2pHmHNefAL4j7iBD4WWkXxYisW0rDLQWzvLeuQ1th4hvp5RN88p/NtbDMEw0ohV3RjyczsF2kJHnHPDz2qb5L5IF4SbdyEb47aW/rQVELxK9FFfVDNPd3e5gskg2PDkLTjdyPc43+7Cco5ogFST+dvcr/YTC3or3 
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDvh9zcxG7a6qATxCCzhQXWjtCCsjALwAiUmwamvilDIL+rL3sRVJjm/aixv9Wi3QTaU/529XkAxkxNVMKo93q6j7wk/Lm2sJR/gdbvxY1IYdz0MdPR1iuM2gSXHaSPvCa+sZ/D4S4JZlrpGab8BM2XJNoTlRH3lNbDu65NRMTk/Ja6z5wZMK8w2pHmHNefAL4j7iBD4WWkXxYisW0rDLQWzvLeuQ1th4hvp5RN88p/NtbDMEw0ohV3RjyczsF2kJHnHPDz2qb5L5IF4SbdyEb47aW/rQVELxK9FFfVDNPd3e5gskg2PDkLTjdyPc43+7Cco5ogFST+dvcr/YTC3or3
```

Comment:
you should probably also get rid of this whitespace change. _probably_ not a big deal, but better to only change what you need to add your key